src/plugins/vis_types/timeseries/public/application/components/aggs/series_agg.js

```diff
@@ -32,6 +34,11 @@ function SeriesAggUi(props) {
 
   const htmlId = htmlIdGenerator();
 
+  const { euiTheme } = useEuiTheme();
+  const titleStyles = css`
+    margin-top: -${euiTheme.size.xs};
+  `;
```

Comment:
This same style is used three different times in this PR. Can likely be consolidated in the common.styles.ts file.

Reply:
done